js/_website/src/routes/[[version]]/docs/[doc=valid_doc]/+page.svelte

```diff
@@ -166,11 +193,19 @@
 							<div class="embedded-component">
 								{#key obj.name}
 									{#if obj.name !== "State"}
-									<gradio-app
-										space={"gradio/" +
-											obj.name.toLowerCase() +
-											"_component_main"}
-									/>
+										{#if on_main}
```

Comment:
nit: I think this can be one line?

```js
space={"gradio/" + obj.name.toLowerCase() +"_component" +  (on_main ? "_main" : "")}
```

Reply:
I actually had it this way at first but it would refuse to update without a refresh. Maybe @pngwn knows why?

Reply:
I think it might be related to changing the space attribute inside gradio-app (vs unmounting and mounting a new one)

Reply:
Ok!